Tools/Scripts/webkitdirs.pm

```diff
@@ -645,8 +646,28 @@ sub parseAvailableXcodeSDKs($)
     return @result;
 }
 
+sub unversionedSDKNameFromSDK($)
+{
+    my $basename = shift;
+    if ($basename =~ /(\D+)(\d+\.[\d\.]+)(\D*)\.sdk/) {
+        if ($3) {
+            return lc "$1.$3";
+        } else {
+            return lc "$1";
+        }
+    }
+}
+
 sub availableXcodeSDKs
 {
+    # Looking for SDKs in known locations is much faster than calling through to xcodebuild.
```

Comment:
Is there a radar for making it faster? Maybe it would also explain why it has to be slow, and what's wrong with this optimization :)

Reply:
Filed rdar://98220832.

Reply:
Thank you. It would be optimal to have a link in the comment, so that anyone (an Apple employee) could immediately see if the workaround was still necessary in a few years from now.